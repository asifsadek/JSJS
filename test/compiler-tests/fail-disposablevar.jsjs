val _ = "a string";
print(_); // should fail since _ is not persisted
