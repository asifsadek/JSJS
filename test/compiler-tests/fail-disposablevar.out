Error: value '_' was used before it was defined

